libs/sdk-ui-ext/src/internal/utils/embeddingCodeGenerator/getWebComponentsCodeGenerator.ts

```diff
@@ -0,0 +1,34 @@
+// (C) 2023 GoodData Corporation
+import { IInsight } from "@gooddata/sdk-model";
+import { IWebComponentsOptions } from "@gooddata/sdk-ui-kit";
+
+const HOST_NAME = "<hostName>";
+const WORKSPACE_ID = "<workspaceId>";
+const INSIGHT_TITLE = "<insightTitle>";
+const INSIGHT_ID = "<insightId>";
+const LOCALE = "<locale>";
+const CUSTOM_HEIGHT = "<customHeight>";
+const CUSTOM_TITLE = "Custom insight title";
+const WEB_COMPONENTS_EMBED_CODE = `<!-- Load the library... -->
+<script type="module" src="https://${HOST_NAME}/components/${WORKSPACE_ID}?auth=sso"></script>
+<!-- ...and embed an insight -->
+
+<gd-insight\n    insight="${INSIGHT_ID}"${INSIGHT_TITLE}${LOCALE}${CUSTOM_HEIGHT}
+></gd-insight>`;
+
+export const getWebComponentsCodeGenerator = (
```

Comment:
Missing unit test for this method.

Reply:
I added the unit test for this method. Thank you!